core/src/templates/components/card/card.twig

```diff
@@ -23,7 +23,19 @@
  * - card_button_label: The text label of the button component.
  */
 #}
-<div{{ attributes }} class="su-card {{ modifier_class }}">
+
+{# For variant where whole card is a link, change the outer wrapper from <div> to <a> #}
+{% if "su-card--link" in modifier_class %}
```

Comment:
This seems overly complex and is a breaking change as the wrapper element can be different. As another possible approach, could we add an optional `<a>` immediately after the wrapper div that is 100% height and width of the card?

Reply:
I can try out this approach.

Reply:
@sherakama , I modified it using your approach - ready for another look. Thanks! Tested on IE, Edge, and the others.